docs/websites/README.md

```diff
@@ -90,6 +90,22 @@ docker logs -f {Docker Container Name}
 
 Refer to **Port lookup table** above for Docker container names
 
+---------
+
+## Website Dependencies
+
+The table below shows server-side dependencies for each website
+
+|Website               | Depends On           |
+|----------------------|-------------         |
+| s.keyman.com         | <br><br>             |
+| downloads.keyman.com | <br><br>             |
+| api.keyman.com       | downloads.keyman.com<br>s.keyman.com |
+| keymanweb.com        | api.keyman.com       |
```

Comment:
```suggestion
| keymanweb.com        | api.keyman.com<br>s.keyman.com |
```

The site fetches both the current version of Keyman Engine for Web and the keyboards via s.keyman.com.

Reply:
Also, if you want to get extra-fun, `r.keymanweb.com` is used for the bookmarklet feature seen on desktop devices.  That's not core page functionality, but it _is_ there.

Reply:
This documentation is about server side dependencies, not client side.

Reply:
Ah.  🤦   That tracks.  It still is due to the dependency chain, but I suppose we don't need to indicate that explicitly.  (keymanweb.com -> api.keyman.com -> s.keyman.com)

Reply:
That said... r.keymanweb.com might still be worth noting:

https://github.com/keymanapp/keymanweb.com/blob/18db526203afa6d9f1eaf8c3e46956e73acb0f92/keyboard-styling-tool.php#L3-L8

Reply:
keyboard-styling-tool is so out of date it is irrelevant -- it doesn't work and should probably just be deleted.

The dependency chain is visible in the documentation; we don't need to drag it out for each site.

Reply:
> keyboard-styling-tool is so out of date it is irrelevant -- it doesn't work and should probably just be deleted.

I did have that feeling... maybe worth making into an issue?

Reply:
> I did have that feeling... maybe worth making into an issue?

or just delete it, it's only a few disconnected files: https://github.com/keymanapp/keymanweb.com/pull/143